package.json

```diff
@@ -2,7 +2,7 @@
   "name": "binary-studio-academy-lecture-abstracts",
   "private": true,
   "description": "Lecture Abstracts for Binary Studio Academy",
```

Comment:
Let's remove all unnecessary items like, description (add it to root readme as h1), version, author, contributors, keywords, license (also remove the license file from the root), repository, bugs

Reply:
What about this?